src/gui/src/viewer/zoom-window.cpp

```diff
@@ -432,7 +433,7 @@ void ZoomWindow::display(const QPixmap &pix, int size)
 		updateWindowTitle();
 
 		if (m_isFullscreen && m_fullScreen != nullptr && m_fullScreen->isVisible()) {
-			m_fullScreen->setImage(m_displayImage.scaled(QApplication::desktop()->screenGeometry().size(), Qt::KeepAspectRatio, Qt::SmoothTransformation));
+			m_fullScreen->setImage(m_displayImage.scaled(QGuiApplication::primaryScreen()->geometry().width(), QGuiApplication::primaryScreen()->geometry().height(), Qt::KeepAspectRatio, Qt::SmoothTransformation));
```

Comment:
You could have just used `geometry().size()` to save some duplication on the call here.

Reply:
Replaced with size()